packages/telestion-client-cli/src/model/index.mts

```diff
@@ -0,0 +1 @@
+export * from './commands.mjs';
```

Comment:
Potentially missing an export from `'./parcel-events.mts'`?

Reply:
Done in [`9c15928` (#1396)](https://github.com/wuespace/telestion-client/pull/1396/commits/9c15928bbbf0d1f30a7c72ca7d7b081c3d878fb6)